Certificate-Sender/.env.sample

```diff
@@ -0,0 +1,17 @@
+# Mail Setup
+EMAIL_ID = "abc1234@gmail.com"  # ADD MAIL ID
+EMAIL_PASSWORD = "xyz@1234"  # ADD PASSWORD
+EMAIL_SUBJECT = "🥳 Thank You For Attending! Here is your Certificate 🎉"  # EMAIL SUBJECT
+CERTIFICATE_EMAIL_BODY = "Hi!
+
+Thanks for your active participation in our webinar on \"All About Git, GitHub & Open Source\" Organized by Dhanraj.
+So now your wait is over and here is your certificate of participation in the webinar.
+Also, don't forget to tag on LinkedIn with your certificate. we are happy to see you there!
```

Comment:
I have clearly mentioned in this suggestion that takes these values as input from user via input function: https://github.com/avinashkranjan/Amazing-Python-Scripts/pull/1131#discussion_r641585677